src/components/Home/Home.js

```diff
@@ -38,6 +38,7 @@ class Home extends Component {
   logout = () => {
     this.props.resetMenuState()
     auth.logout()
+    auth.login()
```

Comment:
I'd make a comment here saying that this redirects the user to the auth0 login form.